### PR TITLE
add angular icon

### PR DESCRIPTION
**Pull request by nyrosmith**
Associated Issue: #3596 

### Summary of Changes

* added existing ng svg to List
* replaced throwing error in dev mode with console.warn

### Test Plan

1. open http://todomvc.com/examples/angularjs/#/
2. set a breakpoint
3. verify that icon gets displayed

### CODE REVIEW

File: assets/images/Svg.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const { isDevelopment } = require("devtools-config");
 
 const svg = {
   "angle-brackets": require("./angle-brackets.svg"),
+  angular: require("./angular.svg"),
```
Comment (jasonLaster) [inline assets/images/Svg.js]: looks like the SVG wasn't included.

File: assets/images/Svg.js [hunk 2]
```diff
@@ -55,12 +56,9 @@ const svg = {
 module.exports = function(name, props) {
   // eslint-disable-line
   if (!svg[name]) {
-    const error = "Unknown SVG: " + name;
     if (isDevelopment()) {
-      throw new Error(error);
```
Comment (jasonLaster) [inline assets/images/Svg.js]: this is okay. It's fine to throw an error in development. The problem is if we try to do it in production. The old version threw an error all the time.

File: assets/images/Svg.js (hunk 1 above)
Comment (nyrosmith) [inline assets/images/Svg.js]: There is an existing angular.svg in the /assets folder which looked good to me.
Shall I replace it with the one you mentioned?

File: assets/images/Svg.js (hunk 2 above)
Comment (nyrosmith) [inline assets/images/Svg.js]: So shall I just revert this part and throw an error in dev mode & console.log?

File: assets/images/Svg.js (hunk 1 above)
Comment (jasonLaster) [inline assets/images/Svg.js]: oh no, i didn't realize that!

File: assets/images/Svg.js (hunk 2 above)
Comment (jasonLaster) [inline assets/images/Svg.js]: yeah - lets revert this part

**Comment by nyrosmith** [comment]
@jasonLaster Just reverted the err throwing behavior as you suggested

**Pull request merged**